pennylane/tape/tape.py

```diff
@@ -96,17 +55,21 @@ def get_active_tape():
     return QueuingManager.active_context()
 
 
-def expand_tape(tape, depth=1, stop_at=None, expand_measurements=False):
+def expand_tape(qscript, depth=1, stop_at=None, expand_measurements=False):
```

Comment:
Why not rename (and move to the script file)? The name is confusing right now.

Reply:
Adding a "TODO" for moving this function to its own file and renaming it.